pyext/src/exhaust.py

```diff
@@ -30,9 +30,12 @@ def parse_args():
         choices=['cuda', 'cpu_omp', 'cpu_serial'], default="cuda")
     parser.add_argument(
         '--cores', '-c', dest="cores", type=int,
-        help='number of cores for parallel clustering at '
-             'different thresholds and RMSD matrix calculations; '
+        help='number of cores for parallel RMSD matrix calculations; '
              'only for cpu_omp', default=1)
+    parser.add_argument(
+        '--cores2', '-c2', dest="cores2", type=int,
+        help='number of cores for clustering at different thresholds'
+             ' and parallel IO; only for cpu_omp', default=1)
```

Comment:
Perhaps you can think of a better name for this argument? `cores2` isn't very descriptive... and short options are usually one character. Maybe `--cluster-cores` (and rename `--cores` to `--matrix-cores`) and `-C` ?

Reply:
Renamed. However, instead of the capital `C` vs a lowercase `c`, I have kept `cc` for cluster cores becuase the capitalization might be more confusing when printed and most other short options are already 2 letters. Let me know if I should shift to `-C`.